topostats/io.py

```diff
@@ -840,49 +885,52 @@ def load_gwy(self) -> tuple:
             image = None
             has_image_found = False
             units = ""
-
-            re = r"\/(\d+)\/data$"
-            components = list(image_data_dict.keys())
-            conv_factors = {"m": 1e9, "um": 1e6, "mm": 1e3}
-            for component in components:
-                match = regex.match(re, component)
-                if match == None:
+            # How can we get current configuration["loading"] ?
+            # default_config.yaml is used to get conf["loading"]["scale"]
+            default_config = pkg_resources.open_text(__package__, "default_config.yaml").read()
+            config = yaml.safe_load(default_config)
+            self.scale = Scale(config["loading"]["scale"])
```

Comment:
With the above suggestion we can move this to the `__init__` method (as PyLint will complain about `self.` being defined outside of `__init__`.